### DISCUSSION TITLE
GFW block by IP

### DISCUSSION THREAD

**Discussion by bubulgum** [shadowsocks/shadowsocks-rust#1038] [Q&A]
Hi all, past 3 month GFW just hard block by IP any server with Shadowsocks-libev and rust. I have tried different providers, different ciphers, but after 1-7 days just hard block by ip. Is there any solution that i have missed, please PM me. For 4 years i have used without any problem....but last 3 month become crazy. For now have only 1 server works, but sometimes ping lost 80% packages, sometime 0. Maybe plugins can help with it?
How to figure out what is the reason?

**Comment by lmmqxyx404** [comment]
You'd better try to use another protocol, like vless or trojan.

**Reply by bubulgum** [reply]
> You'd better try to use another protocol, like vless or trojan.

thx, i will try. but how to find out what is the reason? i can't catch the moment, just suddenly. different locations, different hosts.
